ui/library/src/components/DocumentWrapper.tsx

```diff
@@ -34,7 +34,12 @@ export const DocumentWrapper: React.FunctionComponent<Props> = ({
 
   React.useEffect(() => {
     resetScrollObservers();
-  }, []);
+  }, [pdfDocProxy]);
+
+  React.useEffect(() => {
+    // fix known react-pdf bug where zooming causes scroll position jump to a different page
+    resetScrollToTopOfPage();
```

Comment:
@ericmarsh995 just curious instead of calling this `resetScrollToTopOfPage()` can we call resetScrollObservers() and execute the logic u did inside the `resetScrollToTopOfPage()`